src/scripts/crafttweaker/recipes/trapdoorButtonPlateFix.zs

```diff
@@ -0,0 +1,104 @@
+/*
+	Hacky pressure plate/button/trapdoor crafting fix (removing recipes directly didn't work properly)
+*/
+
+import crafttweaker.item.IItemStack;
+import crafttweaker.item.IIngredient;
+
+import mods.zenstages.Utils;
+
+/*
+    Recipe Removals
+*/
+static removeRecipes as IItemStack[] = [
+	<malisisdoors:trapdoor_spruce>,
+	<malisisdoors:trapdoor_birch>,
+	<malisisdoors:trapdoor_jungle>,
+	<malisisdoors:trapdoor_acacia>,
+	<malisisdoors:trapdoor_dark_oak>
+];
+
+/*
+    Shaped Recipes
+*/
+static shapedRecipes as IIngredient[][][][IItemStack] = {
+	<minecraft:trapdoor> * 2: [
+		[
+			[<minecraft:planks:0> | <chisel:planks-oak:*> | <integrateddynamics:menril_planks> | <rustic:planks:*> | <rustic:painted_wood_white> | <rustic:painted_wood_orange> | <rustic:painted_wood_magenta> | <rustic:painted_wood_light_blue> | <rustic:painted_wood_yellow> | <rustic:painted_wood_lime> | <rustic:painted_wood_pink> | <rustic:painted_wood_gray> | <rustic:painted_wood_silver> | <rustic:painted_wood_cyan> | <rustic:painted_wood_purple> | <rustic:painted_wood_blue> | <rustic:painted_wood_brown> | <rustic:painted_wood_green> | <rustic:painted_wood_red> | <rustic:painted_wood_black> | <twilightforest:tower_wood:*>, <minecraft:planks:0> | <chisel:planks-oak:*> | <integrateddynamics:menril_planks> | <rustic:planks:*> | <rustic:painted_wood_white> | <rustic:painted_wood_orange> | <rustic:painted_wood_magenta> | <rustic:painted_wood_light_blue> | <rustic:painted_wood_yellow> | <rustic:painted_wood_lime> | <rustic:painted_wood_pink> | <rustic:painted_wood_gray> | <rustic:painted_wood_silver> | <rustic:painted_wood_cyan> | <rustic:painted_wood_purple> | <rustic:painted_wood_blue> | <rustic:painted_wood_brown> | <rustic:painted_wood_green> | <rustic:painted_wood_red> | <rustic:painted_wood_black> | <twilightforest:tower_wood:*>, <minecraft:planks:0> | <chisel:planks-oak:*> | <integrateddynamics:menril_planks> | <rustic:planks:*> | <rustic:painted_wood_white> | <rustic:painted_wood_orange> | <rustic:painted_wood_magenta> | <rustic:painted_wood_light_blue> | <rustic:painted_wood_yellow> | <rustic:painted_wood_lime> | <rustic:painted_wood_pink> | <rustic:painted_wood_gray> | <rustic:painted_wood_silver> | <rustic:painted_wood_cyan> | <rustic:painted_wood_purple> | <rustic:painted_wood_blue> | <rustic:painted_wood_brown> | <rustic:painted_wood_green> | <rustic:painted_wood_red> | <rustic:painted_wood_black> | <twilightforest:tower_wood:*>],
```

Comment:
It would be good to use a variable of type IIngredient instead of writing this out for each ingredient in the recipe.

Reply:
I tried to do this at some point, but I guess I did it wrong because I couldn't get it to work. Will be changed shortly.